src/components/cards/Testimonial.jsx

```diff
@@ -0,0 +1,73 @@
+import Image from 'next/image';
+
+export default function Testimonial({
+  position,
+  heading,
+  description,
+  textAlign,
+  clientName,
+  proffession,
+  imagePosition,
+  imageSrc,
+}) {
+  let containerStyles = '';
+  switch (position) {
+    case 'topL':
+      containerStyles =
+        'bg-[#223240] w-full lg:w-[29.375rem] rounded-tl-[80%] shadow-[#2D4850] rounded-tr-[80%] rounded-bl-[100%] mr-[.9375rem]';
```

Comment:
You can define this specifically by extending the TailwindCSS configuration and remove this arbitrary values.